<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevMaintainDetailItemWgt</class>
 <widget class="QWidget" name="DevMaintainDetailItemWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>52</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DevMaintainDetailItemWgt</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="BkFrame">
     <property name="styleSheet">
      <string notr="true">#BkFrame[status=&quot;maintain_wait&quot;]
{
	border-radius:4px;
	background-color:#e6eaf4;
}
#BkFrame[status=&quot;maintain_succ&quot;]
{
	border-radius:4px;
	background-color:#cef5de;
}
#BkFrame[status=&quot;maintain_failed&quot;]
{
	border-radius:4px;
	background-color:#ffdad6;
}
#BkFrame[status=&quot;maintaining&quot;]
{
	border-radius:4px;
	background-color:#cef5de;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="status" stdset="0">
      <string notr="true">maintain_wait</string>
     </property>
     <widget class="QLabel" name="ExeGifLab">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="StatusTextLab">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>5</y>
        <width>91</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>成功</string>
      </property>
     </widget>
     <widget class="QLabel" name="MaintainTimeLab">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>5</y>
        <width>175</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">2023/11/28 00:00:00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReMaintainBtn">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>10</y>
        <width>154</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
color:#025bc7;</string>
      </property>
      <property name="text">
       <string>重新维护</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusIconLab">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>20</y>
        <width>10</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#StatusIconLab[status=&quot;maintain_wait&quot;]
{
	border-radius:5px;
	background-color:#485467;
}
#StatusIconLab[status=&quot;maintain_succ&quot;]
{
	border-radius:5px;
	background-color:#05b30d;
}
#StatusIconLab[status=&quot;maintain_failed&quot;]
{
	border-radius:5px;
	background-color:#e41313;
}
#StatusIconLab[status=&quot;maintaining&quot;]
{
	border-radius:5px;
	background-color:transparent;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="status" stdset="0">
       <string notr="true">maintain_wait</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>153</x>
        <y>5</y>
        <width>420</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="DevNameLab">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="MaintainItemLab">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>光度计检查</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
